game/src/sandbox/gameplay/tutorial.rs

```diff
@@ -1037,24 +1037,25 @@ impl TutorialState {
                     // parking to force the target to park on-street.
                     let map = &app.primary.map;
                     let goal_bldg = map.find_b_by_osm_id(bldg(217701875)).unwrap();
-                    let start_lane = OriginalLane {
-                        parent: OriginalRoad::new(158782224, (1709145066, 53128052)),
-                        num_fwd: 3,
-                        num_back: 3,
-                        dir: Direction::Back,
-                        idx: 0,
-                    }
-                    .from_permanent(map)
-                    .unwrap();
-                    let lane_near_bldg = OriginalLane {
-                        parent: OriginalRoad::new(6484869, (53163501, 53069236)),
-                        num_fwd: 3,
-                        num_back: 3,
-                        dir: Direction::Fwd,
-                        idx: 0,
-                    }
-                    .from_permanent(map)
-                    .unwrap();
+                    let start_lane = {
```

Comment:
This finds a particular road segment by OSM IDs, then selects a specific lane from it. I tested this part of the tutorial to make sure I got the indexing right.